cilium/cmd/service_update.go

```diff
@@ -92,37 +66,40 @@ func boolToInt(set bool) int {
 }
 
 func updateService(cmd *cobra.Command, args []string) {
-	warnIdTypeDeprecation()
-
-	id := int64(idU)
-	fa := parseFrontendAddress(frontend)
 	skipFrontendCheck := false
 
+	fa, err := loadbalancer.NewL3n4AddrFromModelID(frontend)
```

Comment:
This breaks the possibility to update backend state(s)... Usually I do `cilium service update --backends 10.249.97.30:8888 --states maintenance` which means frontend is empty string, but inside the frontend format is checked -> i.e. `Error: Cannot update service: invalid service frontend "", expected "<ip>:<port>:<proto>(:i)" e.g. 1.2.3.4:80:TCP (IPv4, external scope) or [2001::1]:443:TCP:i (IPv6, internal scope)`

Reply:
Refactored this now. Could you take another look?